src/checkPassword.test.js

```diff
@@ -8,12 +8,56 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof checkPassword('false')).toBe('boolean');
   });
 
-  it(`should return 'true' for the valid password with 8 characters`, () => {
+  it(`should return 'false' for the valid password with less than 7 characters`,
```

Comment:
```suggestion
  it(`should return 'false' for the password with less than 7 characters`,
```
якщо він не відповідає вимогам - значить він не валід